File: app/src/main/java/com/mmtran/turtlesoccer/utils/FirebaseSafetyNetUtil.java
package com.mmtran.turtlesoccer.utils;

import android.content.Context;

import com.google.firebase.FirebaseApp;
import com.google.firebase.appcheck.FirebaseAppCheck;
import com.google.firebase.appcheck.safetynet.SafetyNetAppCheckProviderFactory;

public class FirebaseSafetyNetUtil {

    public static void init(Context context) {
        FirebaseApp.initializeApp(context);
        FirebaseAppCheck firebaseAppCheck = FirebaseAppCheck.getInstance();
        firebaseAppCheck.installAppCheckProviderFactory(
                SafetyNetAppCheckProviderFactory.getInstance());
    }
}

File: app/src/main/java/com/mmtran/turtlesoccer/utils/FirebaseSafetyNetUtil.kt
package com.mmtran.turtlesoccer.utils

import android.content.Context
import com.google.firebase.appcheck.FirebaseAppCheck
import com.google.firebase.appcheck.safetynet.SafetyNetAppCheckProviderFactory
import com.google.firebase.FirebaseApp

object FirebaseSafetyNetUtil {
    @JvmStatic
    fun init(context: Context?) {
        FirebaseApp.initializeApp(context!!)
        val firebaseAppCheck = FirebaseAppCheck.getInstance()
        firebaseAppCheck.installAppCheckProviderFactory(
            SafetyNetAppCheckProviderFactory.getInstance()
        )
    }
}
